lib/locales/fr.ts

```diff
@@ -244,7 +251,7 @@ export const locale: Locale = {
     casimir_of_poland: 'Saint Casimir († 1484)',
     catherine_of_alexandria_virgin: 'Sainte Catherine d’Alexandrie, vierge et martyre († IVème s.)',
     catherine_of_saint_augustine_de_simon_de_longpre_virgin:
```

Comment:
According [English Wikipedia](https://en.wikipedia.org/wiki/Catherine_of_St._Augustine) and [French Wikipedia](), her full:

- religious name is `Mary Catherine of Saint Augustine` (note: religious names seem to be localised, therefore in English `Marie` became `Mary` and the hyphen was removed in comparison with its French version);
- name is `Catherine de Simon de Longpré`;

therefore, the ID should be `mary_catherine_of_saint_augustine_catherine_de_simon_de_longpre_virgin`.

```suggestion
    mary_catherine_of_saint_augustine_catherine_de_simon_de_longpre_virgin:
```